src/include/brain/tf_util.h

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// tf_util.h
+//
+// Identification: src/include/brain/tf_util.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
```

Comment:
Copyright should be 2015-2018. I think you can just run the [formatter](https://github.com/cmu-db/peloton/blob/master/script/formatting/formatter.py) to get all those things right.